cmd/commands/cli/command.go

```diff
@@ -37,31 +37,18 @@ const cliCommandName = "cli"
 
 // NewCommand constructs CLI based Mysterium UI with possibility to control quiting
 func NewCommand() *cli.Command {
-	var di cmd.Dependencies
-
 	return &cli.Command{
 		Name:  cliCommandName,
 		Usage: "Starts a CLI client with a Tequilapi",
 		Action: func(ctx *cli.Context) error {
-			errorChannel := make(chan error)
-			if err := di.Bootstrap(cmd.ParseFlagsNode(ctx)); err != nil {
```

Comment:
Is it possible to remove Node bootstrapping for `myst service` command also?

Reply:
But what will the service do without Node?
I think it will be not comfortable to start daemon and service separately.

And we have issue https://github.com/mysteriumnetwork/node/issues/431, that will allow using `service` with existing `daemon`.

Reply:
> But what will the service do without Node?

Well flow should be: `myst daemon --data-dir..` and `myst service`